components/navigation/NavBar.js

```diff
@@ -124,8 +125,11 @@ export default function NavBar({
           >
             <IconLoupe />
           </SearchButton>
-          <button onClick={() => setMobileMenuOpen(true)} type="button" className="inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 focus:text-gray-500 transition duration-150 ease-in-out">
-            <svg className="h-6 w-6" stroke="currentColor" fill="none" viewBox="0 0 24 24">
+          <button onClick={() => {
+            setMobileMenuOpen(true);
+            document.body.style.overflow = 'hidden'
+            }} type="button" className="inline-flex items-center justify-center p-2 text-gray-400 transition duration-150 ease-in-out rounded-md hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 focus:text-gray-500">
```

Comment:
You shouldn't use DOM functions or change the CSS properties using DOM functions directly. Implement some alternative approach.

Reply:
I'm on it🧑🏼‍💻